### PR TITLE
Move X-Plat Components from DNX to CLI

### PR DESCRIPTION
**Pull request by brianrob**
This is a work in-progress PR to validate moving the cross-platform components of xunit.performance to build and run on the .NET CLI.

### CODE REVIEW

**Comment by brianrob** [comment]
@dotnet-bot test Release Build

**Comment by brianrob** [comment]
@ericeil, @dsgouda, @davmason can you please review this?

**Comment by brianrob** [comment]
cc:@lt72

File: src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "version": "99.99.99-dev",
-  "title": "xUnit Performance Runner for DNX",
-  "description": "Contains the tools necessary for running xunit Performance tests on DNX (Windows, Linux, OSX, etc.)",
+  "title": "xUnit Performance Runner for Dotnet CLI",
```
Comment (ericeil) [inline src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json]: Should probably be:

```
xUnit Performance Runner for .NET Core
```

**Comment by ericeil** [comment]
Looks good, aside from one minor naming thing.

**Comment by davmason** [comment]
Looks good to me

File: src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json (hunk 1 above)
Comment (brianrob) [inline src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json]: Yeah, that sounds right to me.  I've made the update.

**Comment by brianrob** [comment]
Thanks much for the review.  I'll merge this once the PR builds complete (and succeed).

**Comment by brianrob** [comment]
@dotnet-bot test Debug Build

**Comment by mmitche** [comment]
@dotnet-bot test Debug Build

**Comment by brianrob** [comment]
@dotnet-bot test Debug Build

**Pull request merged**